### PR TITLE
update Deprecated Domain

### PR DESCRIPTION
**Pull request by max21910**
this domain is not use anymore by Bereal it was removed (can't say why for privacy reason )

### CODE REVIEW

**Comment by max21910** [comment]
add memories  video link

File: README.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ The below contains an overview of the endpoints.
 |----------------------------------------|-------------------------------------------------------------------------------------------------------------------------|-----------------|
 | `https://storage.bere.al/Photos/<uid>` | Where the user generated images are stored, backed by a [GCS Bucket](https://cloud.google.com/storage)                  | `GET HTTP/2.0`  |
 
+| `https://bereal-us-central1-memories.storage.googleapis.com ` | Memories recap video storage
```
Comment (userbradley) [inline README.md:72]: This does not appear to be formatted correctly!

Are you able to change it?

**Review by userbradley — CHANGES_REQUESTED** [review]
Thanks for another PR!

The markdown table does not seem to be formatted correctly. Are you able to fix it?

**Review by max21910 — COMMENTED** [review]
hi i refformated the file sorry for the delay !

**Comment by userbradley** [comment]
Ah sorry @max21910 - If you want to re-open it I'll merge this. For some reason I was not getting notifications from this

**Pull request closed without merging**